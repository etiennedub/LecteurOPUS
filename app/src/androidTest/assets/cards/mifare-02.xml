<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">04930D12</List><List index="1">BA664980</List><List index="2">1548E000</List><List index="3">0000000F</List><List index="4">00010001</List><List index="5">30DF0523</List><List index="6">00000000</List><List index="7">0D59FF3B</List><List index="8">01C3E3C8</List><List index="9">3C803505</List><List index="10">32000000</List><List index="11">2028ABC8</List><List index="12">01C3E3C8</List><List index="13">3E829304</List><List index="14">08000000</List><List index="15">2228EB02</List></Pages></Mifare>