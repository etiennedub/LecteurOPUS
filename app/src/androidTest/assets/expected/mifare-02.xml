<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="18/10/2016" Time="08:20" Transfert="true" OperatorId="20" BusId="41" OperatorName="RTC" BusName="LEBUS 74" /><List index="1" Date="18/10/2016" Time="08:04" Transfert="false" OperatorId="20" BusId="3" OperatorName="RTC" BusName="LEBUS 4" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>